lighthouse-core/report/styles/report.css

```diff
@@ -894,6 +894,9 @@ body {
 .score-poor-bg {
   background-color: var(--poor-color);
 }
+.score-error-bg {
+  background-color: var(--poor-color);
```

Comment:
I chose like...orangish. Based on devtools' alert icon when you do an invalid css property. Happy to change more :)